spyderlib/utils/misc.py

```diff
@@ -11,6 +11,21 @@
 import sys
 import stat
 
+# --- Date/Time
+
+import time
+
+def timestamp(start, end):
+    """Get timestamp like 00:00:02.4950 from start and end times that
+       are collected with time.time()
+    """
+    time_lapse = end - start
+    return (time.strftime("%H:%M:%S.", time.gmtime(time_lapse)) +
+        # gmtime() converts float into tuple, but loses milliseconds
+        ("%.4f" % time_lapse).split('.')[1])

```

Comment:
The function in your last comment should be `time.strftime`, but I see it now, thanks.